umh.docs.umh.app/content/en/docs/getstarted/managingthesystem.md

```diff
@@ -1,142 +1,247 @@
+Then, access Grafana here:
 
-{{% notice info %}}
-Node-RED is a tool, to manage and connect data flows. Simply said, you can 
-specify what data is processed how and then send where. 
+```text
+http://<instance-ip-address>:8080
+```
 
-Each flow starts with an **input node**, where you can specify the data source.
-In this case, the data source is a simulated temperature sensor, the data 
-is conveyed via the MQTT microservice. After the **json node**, which is used to
-parse the data, it is then passed to a **function node**, where you can manipulate
-the data, for example add a timestamp or format it to the correct unit.
-After the manipulation or contextualization, the data is passed on to
-an **output node**, where you can specify the destination of the data.
-{{% /notice %}}
+Use the retrieved credentials to log in.
 
-4. To activate the imported flow, simply click on the **Deploy** button located
-   at the top right of the screen. If everything is working as expected,
-   you should see green dots above the input and output. Once you have confirmed 
-   that the data is flowing correctly, you can proceed to display it in Grafana.
-   ![Untitled](/images/getstarted/managingTheSystem/getStartedManagingDeploy.png?width=75%)
+#### Access the RedPanda Console
 
+The RedPanda Console, used for managing the message broker, can be accessed at:
 
-## 3. Import flows to Grafana & view dashboard
+```text
+http://<instance-ip-address>:8090
+```
 
-1. Go back to the Management Console, close the Node-RED popup and open the
-   Grafana popup. Again, click on the **open** button, to access the Grafana 
-   web interface. To log in, you need the Grafana log in credentials, which you
-   can also find in the Grafana popup.
+#### Interact with the Database
 
-2. Once you are logged in, click on **Dashboards** on the left and select
-   **Import**. Now copy [this Grafana json](/json/getstarted/GrafanaGetStarted.json) 
-   and paste it into **Import via panel json**. Then click on **Load**. You 
-   will then be redirected to **Options** where you need to select the
-   **umh-v2-datasource**. Finally, click on **Import**.
+UMH uses TimescaleDB for database needs. Open a `psql` session with this command:
 
-   ![Untitled](/images/getstarted/managingTheSystem/getStartedManagingGrafanaImport.png?width=75%)
-3. If everything is working properly, you should now see a functional dashboard
-   with a temperature curve.
+```bash
+kubectl exec -it $(kubectl get pods -n united-manufacturing-hub -l app.kubernetes.io/component=timescaledb -o jsonpath="{.items[0].metadata.name}") -n united-manufacturing-hub -- psql -U postgres
+```
 
-   ![Untitled](/images/getstarted/managingTheSystem/getStartedManagingGrafanaDashboard.png?width=75%)
+Run SQL queries as needed. For an overview of the database schema, refer to the
+[Data Model](/docs/architecture/datamodel/database) documentation.
 
```

Comment:
in the future we can also add more sutff like viewieng errors, etc., but it is fine for the ucrrent getting started guide